packages/messenger-widget/src/components/Contacts/Contacts.tsx

```diff
@@ -50,7 +50,7 @@ export function Contacts(props: DashboardProps) {
     // handles contact box view
     useEffect(() => {
         setContactHeightToMaximum(
-            !state.connection.account?.ensName || isAddrEnsName ? true : false,
+            !state.connection.account?.ensName || !isAddrEnsName ? true : false,
```

Comment:
Isn't it sufficient to type !isAddrEnsName?

Reply:
Yes, can be done